gui/CameraView.cxx

```diff
@@ -580,11 +580,13 @@ void CameraView::setLandmarksData(kwiver::vital::landmark_map const& lm)
   }
 
   auto fields = QHash<QString, FieldInformation>{};
-  fields.insert("Elevation", {Elevation, {minZ, maxZ}});
```

Comment:
Does `insert("Elevation", {Elevation, FieldInformation{minZ, maxZ}});` work? I think that would be preferable to needing a named temporary.